src/components/styles/nav.css

```diff
@@ -0,0 +1,248 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600&display=swap');
+
+.nav-container {
+  height: 5.938rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+}
+
+.navtitle a {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+  text-decoration-line: none;
+}
+
+.icon-img {
+  width: 3rem;
+  height: 3rem;
+  margin: 0 0 0 50rem;
+  padding: 0.875rem;
+  border: solid 1px var(--white);
+  color: #0290ff;
+}
```

Comment:
- [ ] Kindly align the icon on the navbar to the right in a straight line with the cards just as it's in the original design. Doing that will make your design looks organized and neat. 👍🏽

<img width="1715" alt="Screenshot 2023-05-27 at 10 10 40 PM" src="https://github.com/emmanuelsteven/BookStore-React-Redux/assets/50941074/4065453b-0421-46af-a95b-c6a59d5be7de">